### PR TITLE
Change from master to main for pipeline generation.

### PR DESCRIPTION
**Pull request by mitchdenny**
This will change the branch that pipeline generation uses from ```master``` to ```main```.

### CODE REVIEW

**Review by benbp — APPROVED** [review]

File: eng/common/pipelines/templates/steps/prepare-pipelines.yml [hunk 1]
```diff
@@ -2,6 +2,9 @@ parameters:
   - name: Repository
     type: string
     default: $(Build.Repository.Name)
+  - name: Branch
```
Comment (weshaggard) [inline eng/common/pipelines/templates/steps/prepare-pipelines.yml:5]: Why not use our default branch template and just query for the default branch before running the tool?

Comment (mitchdenny) [reply eng/common/pipelines/templates/steps/prepare-pipelines.yml]: Swapped over to use it. I also removed the checkout in ```pipeline-generation``` because if we just clone we should pick up the default branch.

File: tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/local.settings.Designer.cs [hunk 2]
```diff
@@ -0,0 +1,242 @@
+﻿//------------------------------------------------------------------------------
```
Comment (weshaggard) [inline tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/local.settings.Designer.cs:1]: This file looks unrelated and probably should be removed.

**Review by weshaggard — APPROVED** [review]

File: eng/pipelines/pipeline-generation.yml [hunk 3]
```diff
@@ -106,27 +100,29 @@ jobs:
   - script: |
       git clone https://github.com/azure/$(RepositoryName) $(Pipeline.Workspace)/$(RepositoryName)
       cd $(Pipeline.Workspace)/$(RepositoryName)
```
Comment (weshaggard) [inline eng/pipelines/pipeline-generation.yml:102]: Can also remove the cd given we aren't running the checkout command.

File: tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/local.settings.Designer.cs (hunk 2 above)
Comment (mitchdenny) [reply tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/local.settings.Designer.cs:1]: Fixed.

Comment (weshaggard) [reply tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/local.settings.Designer.cs:1]: looks like this file is still here.
